minimal-notebook/start-singleuser.sh

```diff
@@ -28,14 +28,21 @@ fi
 
 if [ -n "${JUPYTER_PRELOAD_REPOS}" ]; then
     for repo in `echo ${JUPYTER_PRELOAD_REPOS} | tr ',' ' '`; do
+        # Check for the presence of "#branch" in the repo string
+        REPO_BRANCH=$(echo ${repo} | cut -s -d'#' -f2)
```

Comment:
Is there a particular reason why you chose `#` as a delimiter? I think `pip` uses `@` in `requirements.txt`, not sure what is "more standard way"

Reply:
I can change it to `@`. I only chose `#` because it was the standard used for the same functionality in an old project I worked on.

Reply:
Updated to use `@` as the delimiter